### PR TITLE
#126 - User preferences + Contributors page

### LINKED ISSUE #126: User preferences
A settings menu where user can set all preferences, e.g. Automatic Exam signup

### PR DESCRIPTION
**Pull request by adisve**
Closes #126

### CODE REVIEW

File: App/Data/DataSource/Auth/AuthManager.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ class AuthManager {
     private let urlRequestUtils = NetworkUtilities.shared
     private let keychainManager = KeyChainManager()
     private let urlSession: URLSession = URLSession.shared
+    private let networkController: NetworkController = .shared
```
Comment (ramz1t) [inline App/Data/DataSource/Auth/AuthManager.swift:14]: all three lines 11,13,14 have different initialization and typing 

maybe we can make it the same across the file/project later
private let variable: Class = .shared

File: App/Presentation/Views/Settings/Preferences/PreferenceSettings.swift
Comment (ramz1t) [inline App/Presentation/Views/Settings/Preferences/PreferenceSettings.swift:1]: missing dividers

File: App/Presentation/Views/Settings/SettingsOptions.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    
+    var body: some View {
+        SettingsList {
+            SettingsListGroup {
+                ForEach(allOptions, id: \.id) { option in
+                    SettingsRadioButton(
+                        option: option,
+                        selectedOption: $selectedOption
+                    )
+                    if !(allOptions.last?.id == option.id) {
```
Comment (ramz1t) [inline App/Presentation/Views/Settings/SettingsOptions.swift:22]: Divider shows up even after the last option
image

**Comment by adisve** [comment]
These changes introduce necessary additions to the translation/locales, mentioned in #129

**Review by ramz1t — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**